### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by thanhnga67**

### CODE REVIEW

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (phuongbk) [inline app/helpers/static_pages_helper.rb]: không dùng thì xóa đi :3

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+  	base_title = I18n.t("title")
```
Comment (yuyuvn) [inline app/helpers/application_helper.rb]: indent

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<% provide :title, t("about") %>
+<h1><%= t("about") %></h1>
```
Comment (yuyuvn) [inline app/views/static_pages/about.html.erb]: convention

File: app/views/static_pages/contact.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<% provide :title, t("contact") %>
+<h1><%= t("contact") %></h1>
```
Comment (yuyuvn) [inline app/views/static_pages/contact.html.erb]: convention

File: app/views/static_pages/help.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<% provide :title, t("help") %>
+<h1><%= t("help") %></h1>
```
Comment (yuyuvn) [inline app/views/static_pages/help.html.erb]: convention

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+ <div class="container">
```
Comment (phuongbk) [inline app/views/layouts/_header.html.erb]: indent

File: app/helpers/application_helper.rb [hunk 7]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t("title")
```
Comment (yuyuvn) [inline app/helpers/application_helper.rb]: convention

File: app/helpers/application_helper.rb [hunk 8]
```diff
@@ -1,2 +1,6 @@
...
+    page_title.empty? ? base_title : page_title + "|" + base_title
```
Comment (yuyuvn) [inline app/helpers/application_helper.rb]: don't use + for string, use "#{var}"

**Comment by yuyuvn** [comment]
LGTM

**Pull request merged**